### ISSUE TITLE
Improve WooCommerce Accessibility - Strikethrough Price

### ISSUE THREAD

**Issue by damiannep** [woocommerce/woocommerce#29379]
This enhancement request is to address an accessibility issue that came up in context of WooCommerce Subscriptions but has elements that are generally applicable to WooCommerce in 28028245-hc.

We had a merchant report a legal complaint that their subscription strikethrough price does not meet accessibility standards i.e. there is nothing in the subscription "strikethrough price" that tells screen readers what the two prices mean. There are some specific elements of this request that apply to WooCommerce Subscriptions but the strikethrough price is used generally in WooCommerce so posting the relevant elements here.

Requested alternatives for meeting accessibility needs:

1. Remove the strikethrough price (even if there is a discount applied)
2. Provide ability to prefix the standard price with some text like "Regular price: "
3. Add aria-label (or some other solution) that gives screen readers a meaningful context to what the strikethrough text represents

Raised in Slack in p1614695200122400-slack-C7U3Y3VMY. Visit link for YouTube video showing the issue from merchant's site.

**Comment by mensmaximus** [comment]
Imho the old price should be excluded from screen readers at all (aria-hidden="true") because a strike through price is just a marketing instrument. People with viewing disabilities may get confused hearing two prices because they only want to hear what they have to pay. Consider this scenarios "Save 10%, pay 10$ instead of 11$" or "Old price of $11 is discounted now with 10%. Pay $10 now". Just say "Your price $10".

**Comment by masteradhoc** [comment]
@mensmaximus @juliaamosova added a PR for this. please check if this fixes all occurances of the strikethrough price. thanks!

**Comment by discoinfiltrator** [comment]
> Imho the old price should be excluded from screen readers at all (aria-hidden="true") because a strike through price is just a marketing instrument. People with viewing disabilities may get confused hearing two prices because they only want to hear what they have to pay. Consider this scenarios "Save 10%, pay 10" or "Old price of $11 is discounted now with 10%. Pay $10 now". Just say "Your price $10".

This is ableism and categorically false. If sighted customers are informed that an item is on sale, it is unethical to hide this information form screen reader customers. It strips them of their agency. If a blind customer is confused by the wording then it is a UX concern with how the information is presented, not an issue with the information existing. Removing the information is not an equivalent experience.
